sympy/solvers/ode.py

```diff
@@ -6468,7 +6467,7 @@ def _linear_2eq_order1_type1(x, y, t, r):
 
     """
     l = Dummy('l')
-    C1, C2, C3, C4 = symbols('C1:5')
+    C1, C2, C3, C4 = get_numbered_constants((x(t), y(t)), num=4)
```

Comment:
Would making the `C1, ...` in `sysode_linear_2eq_order1` and pass to these various `_help` helper routines work?